src/experimentEnrollment.js

```diff
@@ -4,20 +4,21 @@ export default {
 
   enrollInExperiment(component, child, variationName) {
     if (variationName && child.props.name === variationName) {
-      component.exposedVariation = child;
+      component.selectedVariation = child;
     } else if (child.props.displayName === DEFAULT_EXPERIMENT_COMPONENT) {
       component.defaultComponent = child;
     }
     return component;
   },
-  
-  getExposedExperimentVariation(childrenComponents, variationName) {
+
+  getVariation(childrenComponents, variationName) {
+    // I don't understand what this line does.
     if (!childrenComponents.reduce) {
```

Comment:
It seems that this should throw an error then (a should be checked via `if(!Array.isArray(childrenComponents))`), since an experiment must necessarily select between more than one component.